protocol/0069-VCBS-validators_chosen_by_stake.md

```diff
@@ -344,6 +344,10 @@ See [limited network life spec](./0073-LIMN-limited_network_life.md).
   * Send an announce node command from a non validator node should fail
 2. Valid announce node command (<a name="0069-VCBS-045" href="#0069-VCBS-045">0069-VCBS-045</a>):
   * Send a valid announce node from a validator node should result in a validator update event with the details of the validator and a validator ranking event.
+3. Node announces using same keys as existing node via announce node command (<a name="0069-VCBS-050" href="#0069-VCBS-050">0069-VCBS-050</a>):
```

Comment:
Let's clarify which keys we're talking about. If any of the vega / tendermint / ethreum key are duplicates of an existing validator or ersatz node then should be rejected.